### ISSUE TITLE
Watermarks with hip

### ISSUE THREAD

**Issue by alexd1971** [lehins/hip#47]
Is it possible to create watermarks with hip library?

**Comment by lehins** [comment]
No, not yet. It definitely would be a really good addition to the library. I have implemented watermarking algorithm before back in grad school, so I don't think it should be too hard. However I am in the process of rewriting hip with newer libraries (see #30 and #40) plus have other commitments, so this will have to wait for a while. Unless you or someone else feels adventurous and would like to submit a PR, I'll be all for it.
